

# CL_KG Summary Report

_Generated: 2025-08-20 13:28 UTC_

---


<details>
<summary><strong>01 — Global totals</strong></summary>


| nodes | relationships |
| --- | --- |
| 277146 | 1243035 |



</details>

---

<details>
<summary><strong>02 — Counts by key label</strong></summary>


| label | n |
| --- | --- |
| Protein | 30972 |
| Biological_process | 25964 |
| Disease | 25804 |
| Cell_cluster | 24850 |
| Gene | 17532 |
| Cell | 10888 |
| Multicellular_anatomical_structure | 10520 |
| Cellular_component | 4058 |
| Race | 927 |
| Developmental_stage | 437 |
| Dataset | 178 |
| Assay | 14 |



</details>

---

<details>
<summary><strong>03 — Average degree by label</strong></summary>


| label | avg_degree |
| --- | --- |
| Dataset | 90.09 |
| Cellular_component | 15.91 |
| Cell_cluster | 15.71 |
| Cell | 13.07 |
| Protein | 8.87 |
| Gene | 8.66 |
| Biological_process | 7.56 |



</details>

---

<details>
<summary><strong>04 — Relationship types by count</strong></summary>


| rt | c |
| --- | --- |
| SUBCLASSOF | 611248 |
| in_subset | 260582 |
| development_stage | 217070 |
| INSTANCEOF | 148096 |
| has_marker | 108116 |
| tissue | 93572 |
| in_taxon | 91611 |
| has_part | 82917 |
| located_in | 53728 |
| sex | 53666 |
| ncbitaxon_has_rank | 48866 |
| subcluster_of | 48330 |
| part_of | 46294 |
| self_reported_ethnicity | 46286 |
| involved_in | 44794 |
| has_participant | 43112 |
| assay | 39084 |
| disease | 37642 |
| has_source | 31364 |
| never_in_taxon | 27254 |
| has_marker_set | 26544 |
| acts_upstream_of_or_within | 24542 |
| is_active_in | 23242 |
| has_role | 20930 |
| produces | 20912 |
| term_replaced_by | 16199 |
| exemplar_data_of | 15794 |
| has_exemplar_data | 15256 |
| chebi_is_conjugate_base_of | 12140 |
| chebi_is_conjugate_acid_of | 12140 |
| has_obsolescence_reason | 11704 |
| chebi_has_functional_parent | 10980 |
| has_primary_input | 10158 |
| composed_primarily_of | 7054 |
| excluded_subClassOf | 6778 |
| present_in_taxon | 6712 |
| consider | 6606 |
| expresses | 6570 |
| regulates | 6286 |
| enables | 6222 |
| positively_regulates | 5530 |
| negatively_regulates | 5458 |
| develops_from | 3768 |
| acts_upstream_of | 3350 |
| crossSpeciesExactMatch | 2842 |
| has_primary_output | 2822 |
| has_characterizing_marker_set | 2746 |
| chebi_is_tautomer_of | 2704 |
| contributor | 2670 |
| has_primary_input_or_output | 2532 |
| chebi_is_enantiomer_of | 2508 |
| capable_of | 2174 |
| obsolete_some_soma_located_in | 1590 |
| mutually_spatially_disjoint_with | 1588 |
| results_in_development_of | 1540 |
| Population_group | 1408 |
| hasCountryOfOrigin | 1382 |
| creator | 1328 |
| contributes_to_morphology_of | 1264 |
| has_plasma_membrane_part | 1204 |
| connects | 1114 |
| lacks_plasma_membrane_part | 1080 |
| has_target_end_location | 1038 |
| occurs_in | 1032 |
| chebi_has_parent_hydride | 942 |
| only_in_taxon | 926 |
| capable_of_part_of | 864 |
| is_opposite_of | 850 |
| has_soma_location | 846 |
| adjacent_to | 810 |
| results_in_morphogenesis_of | 786 |
| overlaps | 778 |
| colocalizes_with | 726 |
| immediately_preceded_by | 712 |
| chebi_is_substituent_group_from | 688 |
| results_in_acquisition_of_features_of | 670 |
| in_historical_homology_relationship_with | 660 |
| results_in_assembly_of | 654 |
| has_target_start_location | 598 |
| results_in_transport_across | 524 |
| connected_to | 460 |
| has_muscle_insertion | 458 |
| isDemonymOf | 456 |
| some_soma_located_in | 452 |
| has_muscle_origin | 438 |
| results_in_formation_of | 432 |
| term_editor | 416 |
| has_potential_to_develop_into | 396 |
| results_in_organization_of | 394 |
| vessel_supplies_blood_to | 376 |
| attached_to | 346 |
| bounding_layer_of | 336 |
| bearer_of | 324 |
| connecting_branch_of | 297 |
| tributary_of | 290 |
| innervated_by | 288 |
| output_of | 278 |
| BSPO_0000126 | 274 |
| extends_fibers_into | 270 |
| partially_overlaps | 268 |
| continuous_with | 262 |
| has_quality | 254 |
| vessel_drains_blood_from | 254 |
| innervates | 238 |
| has_synaptic_IO_in_region | 234 |
| has_high_plasma_membrane_amount | 226 |
| increased_in_magnitude_relative_to | 224 |
| decreased_in_magnitude_relative_to | 222 |
| process_has_causal_agent | 220 |
| has_low_plasma_membrane_amount | 220 |
| results_in_movement_of | 214 |
| has_developmental_contribution_from | 204 |
| surrounds | 190 |
| BSPO_0000120 | 184 |
| has_curation_status | 181 |
| has_relative_magnitude | 180 |
| acts_on_population_of | 178 |
| luminal_space_of | 178 |
| branching_part_of | 176 |
| BSPO_0000121 | 172 |
| has_intermediate | 168 |
| starts_with | 164 |
| has_skeleton | 158 |
| regulates_characteristic | 154 |
| results_in_maturation_of | 154 |
| directly_negatively_regulates | 148 |
| regulates_levels_of | 146 |
| skeleton_of | 142 |
| immediate_transformation_of | 142 |
| results_in_disassembly_of | 140 |
| negatively_regulates_in_another_organism | 138 |
| derived_from_ancestral_fusion_of | 134 |
| results_in_commitment_to | 124 |
| directly_positively_regulates | 122 |
| created_by | 118 |
| proximally_connected_to | 118 |
| participates_in | 114 |
| regulates_in_another_organism | 112 |
| conduit_for | 110 |
| produced_by | 108 |
| results_in_transport_along | 98 |
| directly_regulates | 98 |
| transformation_of | 94 |
| has_muscle_antagonist | 94 |
| dubious_for_taxon | 86 |
| BSPO_0001107 | 80 |
| ends_with | 78 |
| results_in_fusion_of | 78 |
| contributes_to | 78 |
| results_in_structural_organization_of | 74 |
| channel_for | 72 |
| existence_ends_during | 70 |
| surrounded_by | 70 |
| distally_connected_to | 70 |
| results_in_specification_of | 68 |
| temporal_interpretation | 64 |
| results_in_growth_of | 56 |
| acts_upstream_of4_positive_effect | 54 |
| BSPO_0005001 | 54 |
| acts_upstream_of_or_within4_positive_effect | 52 |
| has_start_location | 52 |
| channels_from | 52 |
| results_in_determination_of | 50 |
| preceded_by | 50 |
| contains_process | 48 |
| has_small_molecule_activator | 48 |
| positively_regulates_in_another_organism | 46 |
| has_end_location | 46 |
| results_in_transport_to_from_or_in | 46 |
| lacks_part | 44 |
| anteriorly_connected_to | 44 |
| develops_into | 38 |
| developmentally_induced_by | 38 |
| location_of | 38 |
| subdivision_of | 38 |
| has_member | 38 |
| includedInDataCatalog | 38 |
| channels_into | 36 |
| positively_regulates_characteristic | 34 |
| happens_during | 34 |
| negatively_regulates_characteristic | 34 |
| synapsed_to | 34 |
| BSPO_0000122 | 32 |
| has_not_completed | 30 |
| existence_starts_and_ends_during | 30 |
| existence_starts_during | 30 |
| BSPO_0000123 | 30 |
| derives_from | 28 |
| BSPO_0000098 | 28 |
| BSPO_0000108 | 28 |
| site_of | 26 |
| results_in_maintenance_of | 26 |
| has_stage_marker | 26 |
| has_potential_to_developmentally_contribute_to | 26 |
| axon_synapses_in | 26 |
| has_component | 26 |
| BSPO_0015014 | 26 |
| mmusdv_has_stage_marker | 24 |
| BSPO_0000096 | 24 |
| BSPO_0000102 | 24 |
| BSPO_0000107 | 24 |
| sexually_homologous_to | 24 |
| simultaneous_with | 22 |
| directly_develops_from | 22 |
| BSPO_0001113 | 22 |
| is_count_of | 20 |
| acts_upstream_of4_negative_effect | 20 |
| results_in_remodeling_of | 20 |
| see_also | 18 |
| preceding_element_is | 18 |
| is_homeomorphic_for | 18 |
| results_in_fission_of | 18 |
| synapsed_by | 18 |
| develops_in | 18 |
| BSPO_0000099 | 18 |
| protects | 18 |
| is_positive_form_of | 16 |
| is_negative_form_of | 16 |
| acts_upstream_of_or_within4_negative_effect | 16 |
| results_in | 16 |
| regulated_by | 16 |
| developmentally_replaces | 16 |
| BSPO_0001106 | 16 |
| BSPO_0001100 | 14 |
| BSPO_0020001 | 14 |
| reciprocal_of | 14 |
| results_in_distribution_of | 12 |
| confers_advantage_in | 12 |
| BSPO_0015101 | 12 |
| existence_starts_with | 12 |
| BSPO_0001101 | 12 |
| towards | 12 |
| http___purl_org_dc_terms_contributor | 12 |
| BSPO_0015102 | 10 |
| has_small_molecule_regulator | 10 |
| existence_ends_with | 10 |
| BSPO_0001108 | 10 |
| BSPO_0001115 | 10 |
| develops_from_part_of | 10 |
| has_cross_section | 10 |
| is_direct_form_of | 10 |
| precedes | 8 |
| immediately_precedes | 8 |
| functionally_related_to | 8 |
| coincident_with | 8 |
| BSPO_0000100 | 8 |
| ambiguous_for_taxon | 8 |
| anastomoses_with | 8 |
| filtered_through | 8 |
| in_innermost_side_of | 8 |
| in_outermost_side_of | 8 |
| attached_to_part_of | 8 |
| indirectly_supplies | 8 |
| nominally_disjoint_with | 6 |
| has_synaptic_terminal_in | 6 |
| starts | 6 |
| realizes | 6 |
| hasAncestryStatus | 6 |
| serially_homologous_to | 6 |
| results_in_developmental_progression_of | 4 |
| immediately_causally_upstream_of | 4 |
| results_in_changes_to_anatomical_or_cellular_structure | 4 |
| directly_develops_into | 4 |
| pr_has_constituent_monomer | 4 |
| member_of | 4 |
| has_no_connections_with | 4 |
| ends | 4 |
| characteristic_of | 4 |
| BSPO_0000097 | 4 |
| lumen_of | 4 |
| BSPO_0000124 | 4 |
| BSPO_0000125 | 4 |
| is_indirect_form_of | 4 |
| has_scope | 2 |
| capable_of_regulating | 2 |
| ends_during | 2 |
| involved_in_regulation_of | 2 |
| has_potential_to_directly_develop_into | 2 |
| has_sensory_dendrite_in | 2 |
| has_presynaptic_terminal_in | 2 |
| has_dendrite_location | 2 |
| fasciculates_with | 2 |
| correlated_with | 2 |
| non_functional_homolog_of | 2 |
| pr_has_gene_template | 2 |
| BSPO_0015202 | 2 |
| BSPO_0015009 | 2 |
| BSPO_0015012 | 2 |
| input_of | 2 |
| existence_ends_during_or_before | 2 |
| has_2D_boundary | 2 |
| existence_starts_during_or_after | 2 |
| posteriorly_connected_to | 2 |
| trunk_part_of | 2 |
| layer_part_of | 2 |
| definition_source | 2 |



</details>

---

<details>
<summary><strong>05 — Top tissues across clusters</strong></summary>


| tissue | clusters |
| --- | --- |
| primary motor cortex | 2308 |
| hypothalamus | 2179 |
| primary visual cortex | 1586 |
| primary somatosensory cortex | 1459 |
| hippocampal formation | 1201 |
| anterior cingulate cortex | 1198 |
| cerebellum | 1160 |
| brain | 1131 |
| white matter | 1075 |
| primary auditory cortex | 1074 |
| dorsolateral prefrontal cortex | 1004 |
| midbrain | 975 |
| middle temporal gyrus | 952 |
| striatum | 948 |
| hindbrain | 892 |
| neocortex | 810 |
| pallidum | 791 |
| blood | 725 |
| ventricular system of brain | 724 |
| thalamic complex | 650 |



</details>

---

<details>
<summary><strong>06 — Top diseases across clusters</strong></summary>


| disease | clusters |
| --- | --- |
| normal | 14106 |
| COVID-19 | 718 |
| dementia | 314 |
| Crohn disease | 241 |
| Pick disease | 182 |
| Alzheimer disease | 182 |
| progressive supranuclear palsy | 182 |
| trisomy 18 | 141 |
| chronic kidney disease | 125 |
| acute kidney failure | 124 |
| respiratory system disorder | 106 |
| interstitial lung disease | 93 |
| pulmonary fibrosis | 91 |
| lung adenocarcinoma | 86 |
| non-specific interstitial pneumonia | 85 |
| arrhythmogenic right ventricular cardiomyopathy | 85 |
| dilated cardiomyopathy | 85 |
| chronic rhinitis | 84 |
| hypersensitivity pneumonitis | 82 |
| chronic obstructive pulmonary disease | 82 |



</details>

---

<details>
<summary><strong>07 — Assay types across clusters</strong></summary>


| assay | clusters |
| --- | --- |
| 10x 3&#39; v3 | 7012 |
| 10x 3&#39; v2 | 3290 |
| sci-RNA-seq3 | 1854 |
| MERFISH | 1483 |
| 10x 3&#39; v1 | 1000 |
| Drop-seq | 934 |
| Smart-seq v4 | 820 |
| 10x 5&#39; v2 | 704 |
| 10x 5&#39; v1 | 533 |
| Patch-seq | 347 |
| Smart-seq2 | 333 |
| 10x multiome | 314 |
| snmC-Seq2 | 292 |
| Seq-Well | 158 |
| 10x 5&#39; transcription profiling | 152 |
| 10x 3&#39; transcription profiling | 112 |
| 10x scATAC-seq | 42 |
| snm3C-seq | 41 |
| mCT-seq | 40 |
| Smart-seq | 37 |
| TruDrop | 25 |
| scATAC-seq | 10 |
| Smart-seq3 | 9 |



</details>

---

<details>
<summary><strong>08 — Development stages across clusters</strong></summary>


| stage | clusters |
| --- | --- |
| 29-year-old stage | 3910 |
| 50-year-old stage | 3868 |
| 42-year-old stage | 3818 |
| 60-year-old stage | 3336 |
| 43-year-old stage | 2651 |
| 9-week-old stage | 2323 |
| 8-week-old stage | 1868 |
| 16th week post-fertilization stage | 1842 |
| 15th week post-fertilization stage | 1589 |
| 17th week post-fertilization stage | 1413 |
| 12th week post-fertilization stage | 1411 |
| sixth decade stage | 1395 |
| 13th week post-fertilization stage | 1371 |
| 2-month-old stage | 1367 |
| 65-year-old stage | 1265 |
| 58-year-old stage | 1256 |
| 59-year-old stage | 1239 |
| 70-year-old stage | 1199 |
| 69-year-old stage | 1194 |
| 7-week-old stage | 1156 |
| 56-year-old stage | 1153 |
| 63-year-old stage | 1148 |
| 52-year-old stage | 1126 |
| 4-week-old stage | 1081 |
| 57-year-old stage | 1075 |
| 53-year-old stage | 1056 |
| 68-year-old stage | 1053 |
| 30-year-old stage | 1041 |
| 64-year-old stage | 1035 |
| 75-year-old stage | 1004 |
| 66-year-old stage | 991 |
| 25-year-old stage | 979 |
| 47-year-old stage | 942 |
| 62-year-old stage | 936 |
| 72-year-old stage | 917 |
| 41-year-old stage | 914 |
| juvenile stage | 900 |
| 46-year-old stage | 898 |
| 39-year-old stage | 897 |
| 61-year-old stage | 886 |
| 40-year-old stage | 880 |
| 73-year-old stage | 869 |
| 35-year-old stage | 864 |
| prime adult stage | 863 |
| 37-year-old stage | 849 |
| 36-year-old stage | 848 |
| 54-year-old stage | 846 |
| 55-year-old stage | 843 |
| 31-year-old stage | 839 |
| 78-year-old stage | 825 |
| 45-year-old stage | 825 |
| 38-year-old stage | 807 |
| 27-year-old stage | 800 |
| 71-year-old stage | 799 |
| 49-year-old stage | 796 |
| seventh decade stage | 794 |
| 83-year-old stage | 790 |
| 34-year-old stage | 785 |
| 77-year-old stage | 775 |
| 2-week-old stage | 768 |
| 33-year-old stage | 757 |
| 21-year-old stage | 746 |
| fourth decade stage | 725 |
| 80-year-old stage | 703 |
| 67-year-old stage | 701 |
| fifth decade stage | 692 |
| unknown | 688 |
| 48-year-old stage | 682 |
| 82-year-old stage | 675 |
| 24-year-old stage | 662 |
| 74-year-old stage | 651 |
| 76-year-old stage | 644 |
| 80 year-old and over stage | 633 |
| 23-year-old stage | 628 |
| 51-year-old stage | 619 |
| 44-year-old stage | 617 |
| 84-year-old stage | 610 |
| 6-week-old stage | 603 |
| 26-day-old stage | 591 |
| 81-year-old stage | 583 |
| 86-year-old stage | 573 |
| 1-week-old stage | 571 |
| 26-year-old stage | 565 |
| third decade stage | 534 |
| eighth decade stage | 531 |
| adult stage | 528 |
| 19-year-old stage | 503 |
| 88-year-old stage | 459 |
| 32-year-old stage | 454 |
| 85-year-old stage | 443 |
| 22-year-old stage | 432 |
| 87-year-old stage | 422 |
| 79-year-old stage | 419 |
| 20-month-old stage and over | 418 |
| 23-day-old stage | 400 |
| 89-year-old stage | 397 |
| 20-year-old stage | 396 |
| 10th week post-fertilization stage | 393 |
| 28-year-old stage | 370 |
| 10-week-old stage | 353 |
| 10-year-old stage | 316 |
| postnatal stage | 285 |
| 11-week-old stage | 284 |
| 15-year-old stage | 271 |
| 18-year-old stage | 259 |
| newborn stage (0-28 days) | 252 |
| 12-year-old stage | 252 |
| 13-week-old stage | 233 |
| 3-year-old stage | 228 |
| 4-year-old stage | 225 |
| 92-year-old stage | 223 |
| 60-79 year-old stage | 223 |
| 3-month-old stage | 214 |
| 11-year-old stage | 213 |
| pediatric stage | 208 |
| 14th week post-fertilization stage | 206 |
| 16-year-old stage | 204 |
| late adult stage | 189 |
| 13-year-old stage | 188 |
| Carnegie stage 22 | 185 |
| 24-week-old stage | 184 |
| 14-week-old stage | 184 |
| 18-month-old stage | 183 |
| 9th week post-fertilization stage | 178 |
| 90 year-old and over stage | 177 |
| 14-year-old stage | 175 |
| ninth decade stage | 171 |
| 17-year-old stage | 162 |
| 11th week post-fertilization stage | 152 |
| young adult stage | 145 |
| 9-year-old stage | 144 |
| 6-year-old stage | 140 |
| Carnegie stage 19 | 131 |
| 17-week-old stage | 130 |
| child stage (1-4 yo) | 126 |
| juvenile stage (5-14 yo) | 124 |
| 16-week-old stage | 123 |
| Carnegie stage 23 | 122 |
| 18th week post-fertilization stage | 117 |
| 15-week-old stage | 103 |
| 12-week-old stage | 103 |
| 2-year-old stage | 93 |
| Carnegie stage 18 | 92 |
| Carnegie stage 17 | 88 |
| 19-week-old stage | 87 |
| 91-year-old stage | 87 |
| 20-week-old stage | 85 |
| 18-week-old stage | 83 |
| Theiler stage 25 | 81 |
| Theiler stage 23 | 75 |
| Theiler stage 22 | 74 |
| 6-month-old stage | 71 |
| 90-year-old stage | 70 |
| Theiler stage 27 | 69 |
| Theiler stage 21 | 66 |
| 7-day-old stage | 65 |
| 4-day-old stage | 65 |
| 8-year-old stage | 65 |
| 5-year-old stage | 63 |
| 7-year-old stage | 61 |
| 5-week-old stage | 61 |
| Theiler stage 20 | 55 |
| Theiler stage 18 | 53 |
| 12-month-old stage | 51 |
| embryonic stage | 45 |
| 20th week post-fertilization stage | 44 |
| 23-week-old stage | 42 |
| Carnegie stage 21 | 42 |
| 1-month-old stage | 42 |
| 11-month-old stage | 37 |
| 25-week-old stage | 36 |
| 96-year-old stage | 35 |
| 19-month-old stage | 31 |
| 7-month-old stage | 30 |
| 29-week-old stage | 28 |
| 9-month-old stage | 27 |
| middle aged stage | 25 |
| 22-week-old stage | 24 |
| 26-week-old stage | 21 |
| Carnegie stage 09 | 15 |
| Carnegie stage 13 | 15 |
| Carnegie stage 14 | 15 |
| Carnegie stage 11 | 15 |
| Carnegie stage 12 | 15 |
| 1-year-old stage | 14 |
| 16-month-old stage | 10 |
| 21-week-old stage | 9 |
| 8-month-old stage | 8 |



</details>

---

<details>
<summary><strong>09 — How many datasets and clusters with sources?</strong></summary>


| datasets | clusters_with_source |
| --- | --- |
| 177 | 15682 |



</details>

---

<details>
<summary><strong>10 — Top datasets by #clusters</strong></summary>


| dataset | clusters |
| --- | --- |
| HypoMap – a unified single cell gene expression atlas of the murine hypothalamus | 1397 |
| Survey of human embryonic development | 895 |
| Survey of human embryonic development (1 million cells subset) | 895 |
| Single-cell RNA-seq for all cortical &amp; hippocampal regions (10x) | 436 |
| Single-cell RNA-seq for all cortical &amp; hippocampal regions (SMART-Seq v4) | 431 |
| Tabula Sapiens - All Cells | 385 |
| WB_MERFISH_animal1_coronal | 346 |
| WB_MERFISH_animal2_coronal | 346 |
| WB_MERFISH_animal3_sagittal | 346 |
| Dissection: Primary somatosensory cortex (S1) | 332 |
| Dissection: Primary auditory cortex(A1) | 327 |
| All - A single-cell transcriptomic atlas characterizes ageing tissues in the mouse | 322 |
| Dissection: Primary motor cortex (M1) | 319 |
| Dissection: Primary visual cortex(V1) | 315 |
| Dissection: Anterior cingulate cortex (ACC) | 305 |
| Dissection: Dorsolateral prefrontal cortex (DFC) | 303 |
| Dissection: Angular gyrus (AnG) | 302 |
| Dissection: Middle temporal gyrus (MTG) | 297 |
| An integrated transcriptomic and epigenomic atlas of mouse primary motor cortex cell types | 269 |
| DNA Methylation (CGN) Atlas of the Mouse Brain at Single-Cell Resolution | 215 |



</details>

---

<details>
<summary><strong>11 — List all dataset titles alphabetically with row numbers</strong></summary>


| idx | dataset |
| --- | --- |
| 1 | A molecular single-cell lung atlas of lethal COVID-19 |
| 2 | A proximal-to-distal survey of healthy adult human small intestine and colon epithelium by single-cell transcriptomics |
| 3 | A scRNA-seq atlas of immune cells at the CNS borders |
| 4 | A single-cell multi-omic atlas spanning the adult rhesus macaque brain |
| 5 | A transcriptomic atlas of the mouse cerebellum |
| 6 | AIDA Phase 1 Data Freeze v2: Chinese, Indian, Japanese, Korean, Malay, and Thai donors in Japan, Singapore, South Korea, Thailand, and India |
| 7 | Adult mouse cortical cell taxonomy revealed by single cell transcriptomics |
| 8 | Airway |
| 9 | All - A single-cell transcriptomic atlas characterizes ageing tissues in the mouse |
| 10 | All Cells - snRNA-seq |
| 11 | All cell types of human eye |
| 12 | All cells types from scRNA-seq of human primary sclerosing cholangitis patients and healthy controls |
| 13 | All cells types from snRNA-seq of human primary sclerosing cholangitis patients and healthy controls |
| 14 | All cells with electrophysiological and morphological recordings |
| 15 | All cells with electrophysiological recordings |
| 16 | All cells with morphological recordings |
| 17 | All major cell types in adult human retina |
| 18 | All-snATAC-Spatial multi-omic map of human myocardial infarction |
| 19 | All-snRNA-Spatial multi-omic map of human myocardial infarction |
| 20 | Amacrine cells of the human fovea and peripheral retina |
| 21 | An integrated cell atlas of the human lung in health and disease (core) |
| 22 | An integrated cell atlas of the human lung in health and disease (full) |
| 23 | An integrated transcriptomic and epigenomic atlas of mouse primary motor cortex cell types |
| 24 | Autoimmunity PBMCs |
| 25 | B Cell/Plasmablast Sub_clusters of COVID-19 Immune Altas: Integration of 5 public COVID-19 PBMC single-cell datasets |
| 26 | B cells -- CV19 infection, vaccination and HC |
| 27 | Bipolar cells of the human fovea and peripheral retina |
| 28 | Brain non-myeloid cells - A single-cell transcriptomic atlas characterizes ageing tissues in the mouse |
| 29 | BrainAgingSpatialAtlas_Imputed |
| 30 | BrainAgingSpatialAtlas_MERFISH |
| 31 | BrainAgingSpatialAtlas_MERFISH_LPS |
| 32 | BrainAgingSpatialAtlas_snRNAseq |
| 33 | CGE-derived interneurons integrated with 10X sequencing MOp data |
| 34 | COMBAT project: single cell gene expression data from COVID-19, sepsis and flu patient PBMCs |
| 35 | COVID-19 Immune Altas: Integration of 5 public COVID-19 PBMC single-cell datasets |
| 36 | Central Cornea |
| 37 | Circulating Immune cells -- CV19 infection, vaccination and HC |
| 38 | Classical Monocyte sub_clusters of COVID-19 Immune Altas: Integration of 5 public COVID-19 PBMC single-cell datasets |
| 39 | Colon |
| 40 | DCM/ACM heart cell atlas: All cells |
| 41 | DNA Methylation (CGN) Atlas of the Mouse Brain at Single-Cell Resolution |
| 42 | DNA methylation of brain cells -- CG methylation of Excitatory Neurons (snmC-seq) |
| 43 | DNA methylation of brain cells -- CG methylation of Inhibitory and Subcortical Neurons (snm3C-seq) |
| 44 | DNA methylation of brain cells -- CG methylation of Inhibitory and Subcortical Neurons (snmC-seq) |
| 45 | DNA methylation of brain cells -- CG methylation of Non-neuronal Cells (snm3C-seq) |
| 46 | DNA methylation of brain cells -- CG methylation of Non-neuronal Cells (snmC-seq) |
| 47 | DNA methylation of brain cells -- non-CG methylation of Excitatory Neurons (snm3C-seq) |
| 48 | DNA methylation of brain cells -- non-CG methylation of Excitatory Neurons (snmC-seq) |
| 49 | DNA methylation of brain cells -- non-CG methylation of Inhibitory and Subcortical Neurons (snm3C-seq) |
| 50 | DNA methylation of brain cells -- non-CG methylation of Inhibitory and Subcortical Neurons (snmC-seq) |
| 51 | DNA methylation of brain cells -- non-CG methylation of Non-neuronal Cells (snmC-seq) |
| 52 | Discovery (DIS) set of human colorectal tumor: Epithelial |
| 53 | Dissection: Angular gyrus (AnG) |
| 54 | Dissection: Anterior cingulate cortex (ACC) |
| 55 | Dissection: Dorsolateral prefrontal cortex (DFC) |
| 56 | Dissection: Middle temporal gyrus (MTG) |
| 57 | Dissection: Primary auditory cortex(A1) |
| 58 | Dissection: Primary motor cortex (M1) |
| 59 | Dissection: Primary somatosensory cortex (S1) |
| 60 | Dissection: Primary visual cortex(V1) |
| 61 | Evolution of cellular diversity in primary motor cortex of human - non-neurons |
| 62 | Excitatory neurons integrated with 10X sequencing MOp data |
| 63 | Fetal Human Gut (6-11 PCW) |
| 64 | Fetal kidney dataset: full |
| 65 | Frontal cortex samples from C9-ALS, C9-ALS/FTD and age matched control brains |
| 66 | Frontal cortex samples from C9-FTD and age matched control brains |
| 67 | HBCA - epithelial |
| 68 | HBCA - global |
| 69 | HBCA - stroma |
| 70 | Horizontal cells of the human fovea and peripheral retina |
| 71 | Human Astrocytes 10x scRNA-seq |
| 72 | Human CellCards Multi-Study CellRef 1.0 Atlas |
| 73 | Human DA Neurons 10x scRNA-seq |
| 74 | Human Endothelial cells 10x scRNA-seq |
| 75 | Human Human Microglia 10x scRNA-seq |
| 76 | Human Multiple Cortical Areas SMART-seq |
| 77 | Human Non-DA Neurons 10x scRNA-seq |
| 78 | Human Nurr-Negative Nuclei 10x scRNA-seq |
| 79 | Human Nurr-Positive Nuclei 10x scRNA-seq |
| 80 | Human OPC Cells 10x scRNA-seq |
| 81 | Human Oligodendrocytes 10x scRNA-seq |
| 82 | Human kidney cortex snATAC-seq data from donors with and without diabetic kidney disease |
| 83 | Human kidney cortex snRNA-seq data from donors with and without diabetic kidney disease |
| 84 | HypoMap – a unified single cell gene expression atlas of the murine hypothalamus |
| 85 | Ileum |
| 86 | Immunophenotyping of COVID-19 and influenza highlights the role of type I interferons in development of severe COVID-19 |
| 87 | Individual Single-Cell RNA-seq PBMC Data from Arunachalam et al. |
| 88 | Individual Single-Cell RNA-seq PBMC Data from Guo et al. |
| 89 | Individual Single-Cell RNA-seq PBMC Data from Lee et al. |
| 90 | Individual Single-Cell RNA-seq PBMC Data from Schulte-Schrepping et al. |
| 91 | Individual Single-Cell RNA-seq PBMC Data from Wilk et al. |
| 92 | Integrated HBCA |
| 93 | Integrated Single-nucleus and Single-cell RNA-seq of the Adult Human Kidney |
| 94 | Integrated data |
| 95 | Iris |
| 96 | Large-scale single-cell analysis reveals critical immune characteristics of COVID-19 patients |
| 97 | Lens |
| 98 | M cells -- CV19 infection, vaccination and HC |
| 99 | MGE-derived interneurons integrated with 10X sequencing MOp data |
| 100 | Mature kidney dataset: full |
| 101 | Molecular, spatial and projection diversity of neurons in primary motor cortex revealed by in situ single-cell transcriptomics |
| 102 | Multiomics single-cell analysis of human pancreatic islets reveals novel cellular states in health and type 1 diabetes |
| 103 | Non-neuronal cells of the human fovea and peripheral retina |
| 104 | Non-neuronal cells — An Atlas of Gene Regulatory Elements in Adult Mouse Cerebrum. |
| 105 | Occipital cortex samples from C9-ALS, C9-ALS/FTD and age matched control brains |
| 106 | Occipital cortex samples from C9-FTD and age matched control brains |
| 107 | Organoid - Cell Types of the Human Retina and Its Organoids at Single-Cell Resolution |
| 108 | PBMC |
| 109 | PBMCs |
| 110 | Paediatric Human Gut (4-14y) |
| 111 | Photoreceptor cells of the human fovea and peripheral retina |
| 112 | Placenta Infection |
| 113 | Platelet sub_clusters of COVID-19 Immune Altas: Integration of 5 public COVID-19 PBMC single-cell datasets |
| 114 | Retina |
| 115 | Retinal ganglion cells of the human fovea and peripheral retina |
| 116 | Shared and distinct transcriptomic cell types across neocortical areas |
| 117 | Single Cell Sequencing of Human PBMCs in Clonal Hematopoeisis of Indeterminant Potential |
| 118 | Single cell transcriptional and chromatin accessibility profiling redefine cellular heterogeneity in the adult human kidney - ATACseq |
| 119 | Single cell transcriptional and chromatin accessibility profiling redefine cellular heterogeneity in the adult human kidney - RNAseq |
| 120 | Single-cell RNA-seq analysis reveals cell subsets and gene signatures associated with Rheumatoid Arthritis Disease Activity |
| 121 | Single-cell RNA-seq for all cortical &amp; hippocampal regions (10x) |
| 122 | Single-cell RNA-seq for all cortical &amp; hippocampal regions (SMART-Seq v4) |
| 123 | Single-cell RNA-seq of the Adult Human Kidney (Version 1.0) |
| 124 | Single-cell RNA-seq of the Adult Human Kidney (Version 1.5) |
| 125 | Single-cell atlas of peripheral immune response to SARS-CoV-2 infection |
| 126 | Single-cell multi-omics analysis of the immune response in COVID-19 |
| 127 | Single-cell sequencing links multiregional immune landscapes and tissue-resident T cells in ccRCC to tumor topology and therapy efficacy |
| 128 | Single-cell transcriptomes of the human skin reveal age-related loss of fibroblast priming |
| 129 | Single-nucleus RNA-seq of the Adult Human Kidney (Version 1.0) |
| 130 | Single-nucleus RNA-seq of the Adult Human Kidney (Version 1.5) |
| 131 | Single-nucleus transcriptome data from the dlPFC: Human |
| 132 | Single-soma transcriptomics of tangle-bearing neurons in Alzheimer’s disease - Excitatory |
| 133 | Skin |
| 134 | Supercluster: CGE-derived interneurons |
| 135 | Supercluster: Deep layer (non-IT) excitatory neurons |
| 136 | Supercluster: IT-projecting excitatory neurons |
| 137 | Supercluster: MGE-derived interneurons |
| 138 | Supercluster: Non-neuronal cells |
| 139 | Survey of human embryonic development |
| 140 | Survey of human embryonic development (1 million cells subset) |
| 141 | T Cell and NK Cell Subtypes of COVID-19 Immune Altas: Integration of 5 public COVID-19 PBMC single-cell datasets |
| 142 | T cells -- CV19 infection, vaccination and HC |
| 143 | TI epithelial |
| 144 | TI immune |
| 145 | TI stromal |
| 146 | Tabula Sapiens - All Cells |
| 147 | Time-resolved Systems Immunology Reveals a Late Juncture Linked to Fatal COVID-19: Adaptive Cells |
| 148 | Time-resolved Systems Immunology Reveals a Late Juncture Linked to Fatal COVID-19: Innate Cells |
| 149 | Total - Cells of the human intestinal tract mapped across space and time |
| 150 | Trabecular Meshwork and Corneal Scleral wedge |
| 151 | Type I interferon autoantibodies are associated with systemic immune alterations in patients with COVID-19 |
| 152 | UMAP visualization of all 12 datasets |
| 153 | VAL and DIS datasets: Non-Epithelial |
| 154 | Validation (Val) set of human colorectal tumor: Epithelial |
| 155 | WB_MERFISH_animal1_coronal |
| 156 | WB_MERFISH_animal2_coronal |
| 157 | WB_MERFISH_animal3_sagittal |
| 158 | WB_MERFISH_animal4_sagittal |
| 159 | Whole Taxonomy - DLPFC: Seattle Alzheimer&#39;s Disease Atlas (SEA-AD) |
| 160 | Whole Taxonomy - MTG: Seattle Alzheimer&#39;s Disease Atlas (SEA-AD) |
| 161 | ccRCC - Single-cell analyses of renal cell cancers reveal insights into tumor microenvironment, cell of origin, and therapy response |
| 162 | chRCC - Single-cell analyses of renal cell cancers reveal insights into tumor microenvironment, cell of origin, and therapy response |
| 163 | colon epithelial |
| 164 | colon immune |
| 165 | colon stromal |
| 166 | innate T cells -- CV19 infection, vaccination and HC |
| 167 | multiplexed scRNA-seq of 1.2 million PBMCs from adult lupus samples |
| 168 | normal - Single-cell analyses of renal cell cancers reveal insights into tumor microenvironment, cell of origin, and therapy response |
| 169 | nygc multimodal pbmc |
| 170 | scRNA-seq of foveal and parafoveal retina |
| 171 | snRNA-seq data for human cerebella from 9-12 developmental stages |
| 172 | snRNA-seq data for mouse cerebella from 9-12 developmental stages |
| 173 | snRNA-seq of human anterior and posterior hippocampus |
| 174 | snRNA-seq of human retina - all cells |
| 175 | snmCT-seq of human dorsolateral prefrontal cortex (Brodmann area 46) |
| 176 | white matter - all cells |



</details>

---
